### PR TITLE
Refactor main desktop view

### PR DESCRIPTION
**Pull request by imGok**
- Add padding to main view (side bar + chat list) and side view (chat details)
- Change background colors
- Add chat header for desktop with tab name 
- Add expandable tile to chat list 

(Added space with Element)
image

### CODE REVIEW

File: lib/pages/chat_list/chat_list_body.dart [hunk 1]
```diff
@@ -69,6 +71,138 @@ class ChatListViewBody extends StatelessWidget {
                   ActiveFilter.messages,
                 }.contains(controller.activeFilter) &&
                 client.storiesRooms.isNotEmpty;
+            if (PlatformInfos.isDesktop || PlatformInfos.isWeb) {
+              return ExpandablePanel(
```
Comment (hoangdat) [inline lib/pages/chat_list/chat_list_body.dart:75]: Should be separate in other widget.

File: linux/flutter/generated_plugin_registrant.cc [hunk 2]
```diff
@@ -35,6 +36,9 @@ void fl_register_plugins(FlPluginRegistry* registry) {
   g_autoptr(FlPluginRegistrar) flutter_secure_storage_linux_registrar =
       fl_plugin_registry_get_registrar_for_plugin(registry, "FlutterSecureStorageLinuxPlugin");
   flutter_secure_storage_linux_plugin_register_with_registrar(flutter_secure_storage_linux_registrar);
+  g_autoptr(FlPluginRegistrar) flutter_webrtc_registrar =
+      fl_plugin_registry_get_registrar_for_plugin(registry, "FlutterWebRTCPlugin");
+  flutter_web_r_t_c_plugin_register_with_registrar(flutter_webrtc_registrar);
```
Comment (hoangdat) [inline linux/flutter/generated_plugin_registrant.cc:41]: can you explain more about this change?

File: linux/flutter/generated_plugins.cmake [hunk 3]
```diff
@@ -9,6 +9,7 @@ list(APPEND FLUTTER_PLUGIN_LIST
   emoji_picker_flutter
   file_selector_linux
   flutter_secure_storage_linux
+  flutter_webrtc
```
Comment (hoangdat) [inline linux/flutter/generated_plugins.cmake:12]: idem

File: linux/flutter/generated_plugin_registrant.cc (hunk 2 above)
Comment (imGok) [reply linux/flutter/generated_plugin_registrant.cc:41]: it seems to have been generated when I compiled for desktop applications, I'm not sure if it's needed

Comment (hoangdat) [reply linux/flutter/generated_plugin_registrant.cc:41]: Please remove it

File: lib/config/themes.dart [hunk 4]
```diff
@@ -128,6 +128,9 @@ abstract class FluffyThemes {
           onSecondary: brightness == Brightness.light
               ? AppColor.secondaryPrimary
               : AppColor.colorDEEEFF,
+          onPrimaryContainer: brightness == Brightness.light
+              ? Colors.white
+              : const Color.fromARGB(255, 25, 25, 26)
```
Comment (sherlockvn) [inline lib/config/themes.dart:133]: You should convert color to same format, the heximal form

File: lib/pages/chat_list/chat_list_view/chat_list_view_style.dart [hunk 5]
```diff
@@ -0,0 +1,5 @@
+import 'package:flutter/material.dart';
+
+class ChatListViewStyle {
+  static BorderRadius containerBorderRadius = BorderRadius.circular(16);
```
Comment (sherlockvn) [inline lib/pages/chat_list/chat_list_view/chat_list_view_style.dart:4]: You can use like this to take advantage of const
  static const BorderRadius containerBorderRadius = BorderRadius.all(Radius.circular(16));

File: lib/config/themes.dart (hunk 4 above)
Comment (imGok) [reply lib/config/themes.dart:133]: I can't use heximal, flutter only accepts integer in parameter I can convert it to 32 bits if you prefer

**Comment by hoangdat** [comment]
> overflowed issue

IMO, you have too little items in the list

**Comment by hoangdat** [comment]
IMO, you missed my comment: https://github.com/linagora/twake-on-matrix/pull/27#issuecomment-1502739712.

File: linux/flutter/generated_plugin_registrant.cc (hunk 2 above)
Comment (imGok) [reply linux/flutter/generated_plugin_registrant.cc]: done it

File: lib/config/themes.dart (hunk 4 above)
Comment (imGok) [reply lib/config/themes.dart]: done it

**Pull request closed without merging**